src/utils/url.ts

```diff
@@ -1,9 +1,40 @@
 // src/utils/url.ts
+import { BBox } from "../types";
 
-export { buildUrl };
+export { buildUrl, buildNGDUrl };
 
 function buildUrl(api: string, operation: string, params: any) {
   const root = "https://api.os.uk/search/";
   const query = new URLSearchParams(params);
   return root + `${api}/v1/${operation}?` + query;
 }
+
+function buildNGDUrl(
```

Comment:
Again, shoiuld this be in a shared utils folder or in an ngd-specific folder?

Reply:
moved to ngd sub-directory